pensieve/tests/test_experimenter.py

```diff
@@ -0,0 +1,146 @@
+        "changed_on": "2019-08-07T16:02:43.538514Z",
+        "pretty_status": "Created Delivery",
+        "new_status": "Draft",
+        "old_status": null
+      },
+      {
+        "changed_on": "2019-08-07T20:52:06.859236Z",
+        "pretty_status": "Edited Delivery",
+        "new_status": "Draft",
+        "old_status": "Draft"
+      }
+    ]
+  },
+  {
+    "experiment_url": "https://experimenter.services.mozilla.com/experiments/impact-of-level-2-etp-on-a-custom-distribution/",
+    "type": "pref",
+    "name": "Impact of Level 2 ETP on a Custom Distribution",
+    "slug": "impact-of-level-2-etp-on-a-custom-distribution",
+    "public_name": "Impact of Level 2 ETP",
+    "public_description": "This study enables ETP for a known population to observe impacts on usage and revenue",
+    "status": "Live",
+    "client_matching": "Prefs: n/a\r\n\r\nExperiments: none (different G plugin means we'll ignore the main ETP Level 2 experiment)\r\n\r\nAny additional filters:\r\nnormandy.distribution must be one of the following two options:\r\n* isltd-g-aura-001\r\n* isltd-g-001\r\n    \r\nLess than 200k MAU should be targeted with this filtering.",
+    "locales": [],
+    "countries": [],
+    "platform": "All Platforms",
+    "start_date": null,
+    "end_date": null,
+    "population": "100% of Release Firefox 72.0 to 80.0",
+    "population_percent": "100.0000",
+    "firefox_channel": "Release",
+    "firefox_min_version": "72.0",
+    "firefox_max_version": "80.0",
+    "addon_experiment_id": null,
+    "addon_release_url": null,
+    "pref_branch": "default",
+    "pref_name": "privacy.annotate_channels.strict_list.enabled",
+    "pref_type": "boolean",
+    "proposed_start_date": 1580169600000,
+    "proposed_enrollment": null,
+    "proposed_duration": 180,
+    "variants": [
+      {
+        "description": "this is actually the treatment branch (see background links or ask mconnor for clarity)",
+        "is_control": true,
+        "name": "treatment",
+        "ratio": 100,
+        "slug": "treatment",
+        "value": "true",
+        "addon_release_url": null,
+        "preferences": []
+      }
+    ],
+    "changes": [
+      {
+        "changed_on": "2020-01-07T15:35:19.880806Z",
+        "pretty_status": "Created Delivery",
+        "new_status": "Draft",
+        "old_status": null
+      },
+      {
+        "changed_on": "2020-01-07T15:38:15.351745Z",
+        "pretty_status": "Edited Delivery",
+        "new_status": "Draft",
+        "old_status": "Draft"
+      }
+    ]
+  }
+]
+"""  # noqa
+
+
+@pytest.fixture
+def mock_session():
+    session = Mock()
+    session.get.return_value.json.return_value = json.loads(EXPERIMENTER_FIXTURE)
+    return session
+
+
+def test_fetches_url(mock_session):
+    collection = ExperimentCollection.from_experimenter(mock_session)
+    mock_session.get.assert_called_once_with(ExperimentCollection.EXPERIMENTER_API_URL)
+    assert collection
+    assert len(collection.experiments)
```

Comment:
Any reason you wouldn't want to specify this as `== 2`?

Reply:
Lazy; thought we might want to add more examples to the fixture someday.